api/prisma/seed-staging.ts

```diff
@@ -79,6 +79,14 @@ export const stagingSeed = async (
       [jurisdiction.id],
     ),
   });
+  await prismaClient.featureFlags.create({
+    data: featureFlagFactory(
+      'showCloseListingButton',
+      true,
+      'When true, close button is displayed on the listing edit form',
+      [jurisdiction.id],
+    ),
+  });
```

Comment:
I think this is on me for having a bad description in the task. We want the close button showing as the default so if the feature flag doesn't exist on a jurisdiction the button will appear. 

In other words the feature flag should be "hideCloseListingButton" where if it is turned on for a jurisdiction we hide the button. So the inverse will have to happen on ListingFormActions. 

Sorry about the confusion!